www/source/tutorials/getting-started-process-build.html.md

```diff
@@ -13,14 +13,14 @@ For more information on how to natively run Habitat services or how to run servi
 
 To show the portability of Habitat, you will export and run a Habitat service from within a Docker container through the following steps:
 
-1. If you are not in the studio environment, change over to your plan directory and enter the studio.
+1. If you are not in the studio environment, change over to your plan directory and enter the studio.  (Remember to replace `myorigin` with the name of the origin key you created during `hab setup`. For the examples in this tutorial, it will be set to "myorigin")
 
-        cd ~/Plans/mytutorialapp
+        cd ~/plans/mytutorialapp
         hab studio -k myorigin enter
 
 2. Build your mytutorialapp package.
 
-        [1][default:/src:0]# build
+        [1][default:/src:0]# build mytutorialapp
```

Comment:
but there is no `cd` step above.

the previous step has you `hab studio -k myorigin enter` so you're just in `/src`.  

It's likely you're still in the studio though if you're following along from the previous step so I think we should either remove the step to re-enter the studio, add a `cd mytutorialapp` step, or specify what you're building.

Running the commands as they currently are will definitely cause a failure.